GeneratorInterface/Core/src/LHEWeightHelper.cc

```diff
@@ -134,16 +134,16 @@ namespace gen {
     int close = -1;
     for (size_t idx = 0; idx < headerLines_.size(); idx++) {
       std::string line = headerLines_[idx];
-      if (line.find("<weightgroup") != std::string::npos) {
```

Comment:
This was the problem line for Sanghyun's file: if there are the xml escape characters, this code will think the xml is consistent because it will see NO weightgroups. Removing the < and > seems to work, though it may cause problems if there is a comment that contains "weightgroup". Might be safer to put this code as `line.find("<weightgroup") != std::string::npos || line.find("&lt;weightgroup") != std::string::npos`?